text/batch-l2.md

```diff
@@ -0,0 +1,1540 @@
+# Batch Graduation
+
+* **Original Author(s):**: @comcalvi
+* **Tracking Issue**: #
+* **API Bar Raiser**: @
+
+This proposes a new set of L2 constructs for the aws-batch module. 
+
+Existing (experimental) L2 API docs: https://docs.aws.amazon.com/cdk/api/v1/docs/aws-batch-readme.html
+CloudFormation resource reference: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_Batch.html
+
+## Batch Overview
+
+Batch has four resources:
+
+* `ComputeEnvironment`
```

Comment:
Please also explain Instance classes and types (note on naming: in EC2, "instance TYPE" is a combination of "instance CLASS+instance SIZE" ... if here TYPE in Batch means SIZE only, that's a bit confusing)

Reply:
Batch uses the exact same conventions. However, batch allows you to specify either the class or the type (class + size). If you specify only the class, it will choose the size for you based off what it thinks the job will need.

Reply:
There's a new section, `Choosing your Instance Types`, that should address this.